src/elements/content-sidebar/activity-feed/comment/BaseCommentSneaky.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+import * as React from 'react';
+import BaseCommentWrapper from './BaseCommentWrapper';
+import { type BaseCommentWrapperProps } from './types';
+
+import Replies from './Replies';
+
+const BaseCommentSneaky = (props: BaseCommentWrapperProps) => {
+    const {
+        children,
+        hasReplies,
+        currentUser,
+        getAvatarUrl,
+        getUserProfileUrl,
+        getMentionWithQuery,
+        mentionSelectorContacts,
+        translations,
+        isPending,
+        isRepliesLoading,
+        onHideReplies,
+        onReplyCreate,
+        onSelect,
+        onShowReplies,
+        replies,
+        repliesTotalCount,
+        ...rest
+    } = props;
+
+    return (
+        // Seems like Flow doesn't understand rest operator here?
+        // $FlowFixMe
+        <BaseCommentWrapper {...rest}>
+            {hasReplies && replies ? (
+                <Replies
+                    currentUser={currentUser}
+                    // Get a clue, Flow.
+                    // $FlowFixMe
+                    BaseCommentWrapper={BaseCommentWrapper}
+                    getUserProfileUrl={getUserProfileUrl}
+                    getAvatarUrl={getAvatarUrl}
+                    getMentionWithQuery={getMentionWithQuery}
+                    mentionSelectorContacts={mentionSelectorContacts}
+                    translations={translations}
+                    isParentPending={isPending}
+                    isRepliesLoading={isRepliesLoading}
+                    onHideReplies={onHideReplies}
+                    onReplyCreate={onReplyCreate}
+                    onReplySelect={onSelect}
+                    onShowReplies={onShowReplies}
+                    replies={replies}
+                    repliesTotalCount={repliesTotalCount}
+                />
+            ) : (
+                undefined
```

Comment:
Only did this (as apposed to `replies && Replies`) to satisfy Flow. Lmk if you all know the trick :)

Reply:
this ended up working for me: `{hasReplies && replies && <Replies [...] />}`

one of the other `$FlowFixMe` might've fixed the issue you were seeing 🤷